File: EASY/ONBOARDING/clojure/onboarding.clj
(ns Player
  (:gen-class))

(defn -main [& args]
  (while true
    (let [enemy1 (read) dist1 (read) enemy2 (read) dist2 (read)]
      ; enemy1: name of enemy 1
      ; dist1: distance to enemy 1
      ; enemy2: name of enemy 2
      ; dist2: distance to enemy 2

      (if (< dist1 dist2) (println enemy1) (println enemy2))
)))

File: EASY/ONBOARDING/java/onboarding.java
import java.util.*;

class Player
{
    public static void main(String args[])
    {
        Scanner in = new Scanner(System.in);
        while (true)
        {
            String enemy1 = in.next(); // name of enemy 1
            int dist1 = in.nextInt(); // distance to enemy 1
            String enemy2 = in.next(); // name of enemy 2
            int dist2 = in.nextInt(); // distance to enemy 2

            if (dist1 < dist2)
                System.out.println(enemy1);
            else
                System.out.println(enemy2);
        }
    }
}